src/test/java/org/opentripplanner/routing/core/CarRoutingTest.java

```diff
@@ -0,0 +1,83 @@
+package org.opentripplanner.routing.core;
+
+import com.google.common.collect.Lists;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.opentripplanner.ConstantsForTests;
+import org.opentripplanner.api.model.TripPlan;
+import org.opentripplanner.api.resource.GraphPathToTripPlanConverter;
+import org.opentripplanner.common.model.GenericLocation;
+import org.opentripplanner.graph_builder.GraphBuilder;
+import org.opentripplanner.graph_builder.module.StreetLinkerModule;
+import org.opentripplanner.graph_builder.module.osm.GermanyWayPropertySetSource;
+import org.opentripplanner.graph_builder.module.osm.OpenStreetMapModule;
+import org.opentripplanner.graph_builder.services.DefaultStreetEdgeFactory;
+import org.opentripplanner.openstreetmap.impl.AnyFileBasedOpenStreetMapProviderImpl;
+import org.opentripplanner.openstreetmap.services.OpenStreetMapProvider;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.impl.DefaultStreetVertexIndexFactory;
+import org.opentripplanner.routing.impl.GraphPathFinder;
+import org.opentripplanner.routing.spt.GraphPath;
+import org.opentripplanner.standalone.Router;
+import org.opentripplanner.util.TestUtils;
+
+import java.io.File;
+import java.util.List;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class CarRoutingTest {
+
+    static Graph graph;
+
+    static GenericLocation seeStrasse = new GenericLocation(48.59724504108028,8.868606090545656);
+    static GenericLocation offTübingerStr = new GenericLocation(48.58529481682537, 8.888196945190431);
```

Comment:
Perhaps it's old fashioned, but I'd prefer no Umlauts in code

Reply:
It's quite old-fashioned since everything is a Unicode nowadays but I will change it.

Reply:
ok